projects/wvr-elements/src/lib/wvr-footer/wvr-footer.component.spec.ts

```diff
@@ -29,4 +29,17 @@ describe('WvrFooterComponent', () => {
       .toBeFalse();
   });
 
+  it('should positionSelf when window is resized', () => {
+    const spyOnResize = spyOn(component, 'positionSelf');
+    window.dispatchEvent(new Event('resize'));
+    expect(spyOnResize).toHaveBeenCalled();
+
+    const isSticky = false;
+    const newIsSticky = 420 <= (window.innerHeight - 65);
+    expect((isSticky !== newIsSticky)).toBeTruthy();
```

Comment:
These assertions are not referencing the component

Reply:
assertions referencing the isSticky property is now being referenced.